packages/jss-plugin-template/benchmark/tests/createSheet.js

```diff
@@ -4,7 +4,7 @@ import template from '../../src/index'
 import parse from '../../src/parse'
 
 const options = {Renderer: null}
-const jss = create(options).use(template())
+const jss = create(options).use(template({cache: false}))
```

Comment:
disabling cache here to have a realistic comparison without caching, I didn't document it, because I don't think user should need it, so for now it's a private one, just for tests